packages/@okta/vuepress-site/docs/guides/submit-oin-app/main/index.md

```diff
@@ -51,24 +52,28 @@ This guide covers submissions that use the following protocols:
 
 * [Security Assertion Markup Language (SAML)](http://docs.oasis-open.org/security/saml/Post2.0/sstc-saml-tech-overview-2.0.html)
 
-    > **Note:** SAML integrations must use SHA256 encryption for security. If you're using SHA-1 for encryption, see our guide on how to [Upgrade SAML Apps to SHA256](/docs/guides/updating-saml-cert/).
+    > **Notes:**
+    > * SAML integrations must use SHA256 encryption for security. If you're using SHA-1 for encryption, see our guide on how to [Upgrade SAML Apps to SHA256](/docs/guides/updating-saml-cert/).
+    > * The OIN Wizard doesn't support advanced SAML integrations, such as integrations with more than three app instance variables. Some ISVs use many app instance variables to help identify app instances for each of their customers.
```

Comment:
Hm, this sort of makes it sound like our SAML support isn't as good as we'd like it to be, when in reality we're intentionally limiting it for security and usability reasons. Plus there's not really a well-defined concept of "advanced" vs. "basic" SAML integrations even though we've been using it a bit internally recently to refer to our full internal config vs. what we expose in the submission API.

Maybe something like, "The OIN Wizard places certain limits on submissions. For example, submissions are limited to three app instance variables. Some ISVs..."

Reply:
updated based on suggestions